### ISSUE TITLE
HDR support for Stremio Desktop

### ISSUE THREAD

**Issue by whoisarpit** [Stremio/stremio-features#481]
**Is your feature request related to a problem? Please describe.**
Current version doesn't support HDR.

**Describe the solution you'd like**
HDR Support.

**Describe alternatives you've considered**
The play with VLC option, but that pretty much breaks the whole experience of Stremio.

**Additional context**
-

**Comment by TRtomasz** [comment]
Have you checked if it works when you enable hardware accelerated decoding in the settings?

**Comment by whoisarpit** [comment]
@TRtomasz yes turning on hardware accelerated decoding doesn't really change anything

**Comment by TRtomasz** [comment]
now available with stremio 5
